packages/components/src/table/test/index.js

```diff
@@ -73,10 +93,12 @@ describe( 'TableCard', () => {
 
 		// We shouldn't get here if an error occurred.
 		expect( true ).toBe( true );
+
+		expect( container ).toMatchSnapshot();
```

Comment:
We probably don't need a snapshot test here, there is not structural change in this test.

Reply:
😀 Agreed! I noticed this but I went to the side "better safe than sorry" :D 
Addressed 667be04a9eddeefb1ffa6e64b36c7fd8f38e1913